docs/configuration/notifier/index.md

```diff
@@ -36,6 +37,51 @@ The notifier has a startup check which validates the specified provider
 configuration is correct and will be able to send emails. This can be
 disabled with the `disable_startup_check` option:
 
+### template_path
+<div markdown="1">
+type: string
+{: .label .label-config .label-purple }
+default: ""
+{: .label .label-config .label-blue }
+required: no
+{: .label .label-config .label-green }
+</div>
+
+This option allows the administrator to set custom templates for notifications
+the templates folder should contain the following files
+
+|File                    |Description                                        |
+|------------------------|---------------------------------------------------|
+|PasswordResetStep1.html |HTML Template for Step 1 of password reset process |
+|PasswordResetStep1.txt  |Text Template for Step 1 of password reset process |
+|PasswordResetStep2.html |HTML Template for Step 2 of password reset process |
+|PasswordResetStep2.txt  |Text Template for Step 2 of password reset process |
+
+Note:
+* if you don't define some of these files, a default template is used for that notification
+
+
+In template files, you can use the following variables:
+
+|File                    |Description                                        |
+|------------------------|---------------------------------------------------|
+|`{{.title}}`| A predefined title for the email. <br> It will be `"Reset your password"` or `"Password changed successfully"`, depending on the current step |
+|`{{.url}}`  | The url that allows to reset the user password |
+|`{{.displayName}}` |The name of the user, i.e. `John Doe` |
+|`{{.button}}` |The content for the password reset button, it's hardcoded to `Reset` |
```

Comment:
Would be nice to add the RemoteIP. Though I have a feeling this may take some additional work in the SMTP area so I'm fine if you skip that part.

```suggestion
In template files, you can use the following variables:

|File                    |Description                                        |
|------------------------|---------------------------------------------------|
|`{{.title}}`| A predefined title for the email. <br> It will be `"Reset your password"` or `"Password changed successfully"`, depending on the current step |
|`{{.url}}`  | The URL used for identity verification in the email |
|`{{.displayName}}` |The name of the user, i.e. `John Doe` |
|`{{.button}}` |The content for the password reset button, it's hardcoded to `Reset` |
|`{{.remoteIP}}` |The remote IP address that initiated the request or event |
```

Reply:
Hi @james-d-elliott , is there any section where i can read the remote IP from? i can add to the notification.

Reply:
Done!
![imagen](https://user-images.githubusercontent.com/10672208/158685889-2ddd5d86-8f19-491d-9ebc-8de7faab638a.png)

Reply:
Nice one. You did it how I would have.